### PR TITLE
First stage of Tab Discarding doc for @addyosmani

### PR DESCRIPTION
**Pull request by PaulKinlan**
cc @jpmedley as an FYI.  This has been reviewed by the Chrome team and others.

@addyosmani I compressed a bunch of the images, but I am wondering if there is any issue with me halving the size of them (they are pretty massive).

### CODE REVIEW

File: src/_langs/en/updates/2015-09-01-tab-discarding.markdown [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+### Background
+
+For every tab you have open our renderer process usually takes around 50MB per 
+tab, even though most people use just a single tab at a time. If you've got 10 
+tabs open simultaneously there's at least 450MB of memory being spent just to 
+keep your background tab state. This can get a little unwieldy over time.
+
+<img src="/web/updates/images/2015-09-01-tab-discarding/image01.png" />
+
+One of our goals is to reduce the memory used on tabs you're **not **actually
```
Comment (arthurvr) [inline src/_langs/en/updates/2015-09-01-tab-discarding.markdown]: `**not** actually` (spacing)

**Comment by jpmedley** [comment]
@PaulKinlan @addyosmani Copy editing now.

**Comment by addyosmani** [comment]
Thanks for prepping this and copy reviewing. I'll see if I can get the size of images down without sacrificing on readability. That's probably the only issue with half-sizing the larger ones.

**Comment by PaulKinlan** [comment]
I authored the commits.

**Pull request merged**